charts/corda/values.schema.json

```
@@ -3045,6 +3060,15 @@
                         },
                         "additionalProperties": false
                     }
+                },
+                "extraArgs": {
+                    "type": "array",
+                    "default": [],
+                    "title": "Allows users to pass any extra arguments to the worker",
+                    "items": {},
```

this should specify the type of the items in the array:
```suggestion
                    "items": {
                        "type": "string"
                    },
```